js/usernameSelection.js

```diff
@@ -43,19 +42,21 @@ const addUsername = function addUsername() {
 };
 
 const addOneUsername = function addOneUsername(username) {
-    USERNAMES.push(username);
+    let usernames = JSON.parse(localStorage.getItem('usernames'));
+    usernames.push(username);
+    localStorage.setItem('usernames', JSON.stringify(usernames));
     content.innerHTML = '';
     getData();
     renderUsernames();
 };
 
-const inputOptions = new Promise((resolve) => {
-    setTimeout(() => {
-        resolve(USERNAMES);
-    }, 2000);
-});
-
 const removeUsername = function removeUsername() {
+    inputOptions = new Promise((resolve) => {
+        let usernames = JSON.parse(localStorage.getItem('usernames'));
+        setTimeout(() => {
```

Comment:
This looks like it's "simulating" an ajax call (showing a loading animation for 2 seconds, then closing) I think this is _fine_, at least for now. :)
IMO we should not get too fancy here if we're going to almost rewrite everything ni js modules later :D